src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -101,6 +101,18 @@ public partial struct ObjectEnumerator : System.Collections.Generic.IEnumerable<
             System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
         }
     }
+    public readonly partial struct JsonEncodedText : System.IEquatable<System.Text.Json.JsonEncodedText>
+    {
+        private readonly object _dummy;
+        public System.ReadOnlySpan<byte> EncodedUtf8Bytes { get { throw null; } }
+        public static System.Text.Json.JsonEncodedText Encode(System.ReadOnlySpan<byte> utf8Value) { throw null; }
+        public static System.Text.Json.JsonEncodedText Encode(System.ReadOnlySpan<char> value) { throw null; }
+        public static System.Text.Json.JsonEncodedText Encode(string value) { throw null; }
```

Comment:
Rather than introduce a break when the JavascriptEncoder stuff comes online, perhaps make the API take one as a default parameter now, but have the runtime throw if it sees a non-null value.

Reply:
`System.Text.Encodings.Web` is not in-box atm so I can't add it to the signature at the moment.

cc @GrabYourPitchforks